cmd/podman/shared/intermediate_varlink.go

```diff
@@ -0,0 +1,383 @@
+// +build varlink remoteclient
+
+package shared
```

Comment:
Can we get a comment here, and in intermediate_varlink.go, explaining all the duplication (performance and avoiding reflection)?

Reply:
Sorry, the novarlink file as well, I mean